src/compiler/transformers/classFields.ts

```diff
@@ -218,19 +348,47 @@ namespace ts {
         }
 
         function createPrivateIdentifierAccess(info: PrivateIdentifierInfo, receiver: Expression): Expression {
-            receiver = visitNode(receiver, visitor, isExpression);
-            switch (info.placement) {
-                case PrivateIdentifierPlacement.InstanceField:
-                    return context.getEmitHelperFactory().createClassPrivateFieldGetHelper(
-                        nodeIsSynthesized(receiver) ? receiver : factory.cloneNode(receiver),
-                        info.weakMapName
+            return createPrivateIdentifierAccessHelper(info, visitNode(receiver, visitor, isExpression));
+        }
+
+        function createPrivateIdentifierAccessHelper(info: PrivateIdentifierInfo, receiver: Expression): Expression {
+            let getExpression: Expression;

```

Comment:
As mentioned in my other comment, you can do this here:

```suggestion
            setCommentRange(receiver, moveRangePos(receiver, -1))
```
And revert to `return` in each switch branch below.